packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -91,12 +95,20 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
         showCameraSwitcherInLocalPreview={props.isMobile}
         localVideoCameraCycleButtonProps={cameraSwitcherProps}
         onRenderAvatar={onRenderAvatar}
-        /* @conditional-compile-remove(pinned-participants) */
-        showRemoteVideoTileContextualMenu={!props.isMobile}
         // @TODO: Provide props.drawerMenuHostId to VideoGallery when VideoGallery props support it.
+        /* @conditional-compile-remove(pinned-participants) */
+        remoteVideoTileMenuOptions={
+          props.remoteVideoTileMenuOptions?.isHidden ? false : props.isMobile ? false : { kind: 'contextual' }
```

Comment:
Simplify this double ternary. Also when it is mobile assign { kind: 'drawer', hostId: <someHostId> }.